src/GraphQL.Tests/xunit.runner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+  "diagnosticMessages": true,
+  "longRunningTestSeconds": 10
+}
```

Comment:
In the event of a hung test, this allows determination of which test(s) caused the hang.

Reply:
Still needed?

Reply:
I recommend leaving this to be able to diagnose future hung tests.

Reply:
ok

Reply:
Do I need to look into the dough logs, if something hangs?

Reply:
The GitHub Action log will start repeating stuff like this over and over if it hangs.  It will be obvious because it will show the test running for 6 hours (max time) before it terminates the process.

```
[xUnit.net 00:00:36.71] GraphQL.Tests: [Long Running Test] 'GraphQL.Tests.Types.AutoRegisteringObjectGraphTypeTests.ThrowsWhenSourceNull_Struct', Elapsed: 00:00:32
[Long Running Test] 'GraphQL.Tests.Execution.ResolveFieldContextTests.ExecutionError_Should_Be_Thread_Safe', Elapsed: 00:00:34
[Long Running Test] 'GraphQL.Tests.Subscription.SubscriptionWithReflectionTests.SubscribeWithArgumentAsync', Elapsed: 00:00:34
[Long Running Test] 'GraphQL.Tests.Subscription.SubscriptionTests.SubscribeWithArgumentAsync', Elapsed: 00:00:34
[Long Running Test] 'GraphQL.Tests.Bugs.Bug2635.test_parallel', Elapsed: 00:00:34
[Long Running Test] 'GraphQL.Tests.Execution.MultithreadedTests.test', Elapsed: 00:00:33
[Long Running Test] 'GraphQL.Tests.Execution.MutationTests.evaluates_mutations_correctly_in_the_presence_of_a_failed_mutation', Elapsed: 00:00:33
[Long Running Test] 'GraphQL.Tests.Subscription.SubscriptionWithAutoGraphTypeTests.SubscribeAsync', Elapsed: 00:00:30
[Long Running Test] 'GraphQL.Tests.Execution.Cancellation.CancellationTests.cancellation_is_propagated_async', Elapsed: 00:00:30
```